components/ai_chat/resources/page/components/conversations_list/index.tsx

```diff
@@ -94,7 +95,7 @@ function ConversationItem(props: ConversationItemProps) {
           isOptionsMenuOpen && styles.isOptionsMenuOpen
         )}
         onClick={(e) => {
-          if (isEditing) {
+          if (isEditing || isDeleting) {
```

Comment:
shouldn't this button not be clickable because a dialog is open?

Reply:
ahhh yes, I was under the impression we were preventing the `href={`/${uuid}`}` click here. Removed this!